### ISSUE TITLE
Community Meeting January 17th, 2024

### ISSUE THREAD

**Issue by zregvart** [enterprise-contract/community#23]
## Overview
At 10am Eastern, 7am Pacific, 2pm UTC; [find your time](https://www.timeanddate.com/worldclock/converter.html?iso=20210518T140000&p1=1440&p2=4826&p3=234&p4=195)

Video call link: [https://meet.google.com/ved-ivnv-aen](https://meet.google.com/ved-ivnv-aen?hs=122&authuser=0)
dial-in: (US) +1 409-444-2447‬
PIN: ‪873 550 579#‬
More phone numbers: [tel.meet/ved-ivnv-aen?pin=7872592963542&hs=1](https://tel.meet/ved-ivnv-aen?pin=7872592963542&hs=1)

**Add topics you'd like to discuss below!**

**Comment by zregvart** [comment]
:confetti_ball: Happy New Year :tada:

**Comment by zregvart** [comment]
Let's discuss [Backstage](https://backstage.io/) integration.

**Comment by lcarva** [comment]
I'd like to discuss "acceptable bundles" as described [here](https://enterprisecontract.dev/docs/ec-policies/acceptable_bundles.html).

> The list of acceptable bundles is time based. A bundle that is acceptable today is not necessarily acceptable tomorrow. The list below may contain bundles that are "too old" and no longer acceptable. The reason for this behavior is to allow users a certain period of time to upgrade to a newer bundle.

The concept of entries being "too old" is confusing, making it hard to get the implementation right, and even to explain how it is supposed to work for users.

To further complicate things, the complete feature is implemented partially in ec-cli and partially in ec-policies.

Maybe it's time to find an alternate approach?

**Comment by lcarva** [comment]
Bumping this to next week due to low attendance.

**Comment by lcarva** [comment]
We discussed the backstage integration topic. Backstage is mostly a view aggregator. In order to have EC integration, we would likely need to first have EC as a service. Next step is to create a proof of concept of what EC as a service would look like. Then, we can revisit this topic and likely have another proof of concept backstage plugin.

Moved unaddressed topics to next week https://github.com/enterprise-contract/community/issues/24.
